recipes/tximport-scripts/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: tximport-scripts
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/tximport-scripts/archive/v{{ version }}.tar.gz 
+  sha256: 7910a7519dd6655d8a2007019639d4204829aeb8d1a9f6a94d0c80055003ea10 
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+    run:
+        - r-base 3.5.1
```

Comment:
you can remove the pinning here, we do this automatically now.

Reply:
Okay, thanks @bgruening , I've changed that. BTW, any idea why the CI is getting stuck in the queue for the macos part but not the linux?

Reply:
We've contacted circleci about that, we're waiting to hear back from them.